packages/react-components/react-popover/src/components/PopoverSurface/usePopoverSurface.ts

```diff
@@ -48,7 +48,7 @@ export const usePopoverSurface_unstable = (
       ref: useMergedRefs(ref, contentRef),
       role: trapFocus ? 'dialog' : 'group',
       'aria-modal': trapFocus ? true : undefined,
-      ...modalAttributes,
+      ...(trapFocus ? modalAttributes : {}),
```

Comment:
can you explain the motivation behind this change? I'm trying to understand if there is a bug on tabster here. The modal attributes already allow focus trapping to be configured.

If there is an problem with applying these attributes with focus trap, please create an issue for it and attach it to this PR so we can investigate and determine the tabster issue

Reply:
Sorry for the late reply @ling1726, the problem this is trying to solve is that when we have a popover that shouldn't trap focus and is not rendered in a portal, it should be able to move back and forth from the popover to the rest of the dom. This is not possible because the modal attributes are being applied even if the popover is no modal. I understand that the modal attributes need to be applied when trapping focus as it makes it modal, but when we don't trap focus, these should not be applied.

Reply:
Sorry I still can't see the problem here, if trapFocus is not set, then tabster does not implement a focus trap. Do you mind creating a quick sandbox to demo the problem so we can understand if there is a lower level bug in tabster that needs to be addressed as well?